website/sidebars.js

```diff
@@ -468,7 +469,7 @@ const sidebarSettings = {
         "docs/collaborate/collaborate-with-others",
         {
           type: "category",
-          label: "Explore dbt projects",
+          label: "Discover projects with dbt Explorer",
```

Comment:
@cafzal how do we feel about this new sidebar label to further improve discoverability? suggested by katherine.